types/node/process.d.ts

```diff
@@ -793,6 +906,12 @@ declare module "process" {
                  * @since v17.3.0, v16.14.0
                  */
                 getActiveResourcesInfo(): string[];
+                /**
+                 * Provides a way to load built-in modules in a globally available function.
+                 * @param id ID of the built-in module being requested.
+                 */
+                getBuiltinModule<ID extends keyof BuiltInModule>(id: ID): BuiltInModule[ID];
```

Comment:
This is also available in 20.16; should that be updated too?

Reply:
Yes, but I think we can make separate PRs for v20.15.x and v20.16.x